packages/core/content-manager/shared/contracts/collection-types.ts

```diff
@@ -324,7 +324,7 @@ export declare namespace BulkPublish {
 export declare namespace BulkUnpublish {
   export interface Request {
     body: {
-      ids: number[];
+      documentIds: number[];
```

Comment:
```suggestion
      documentIds: string[];
```

?

Reply:
yes, they're actually all different which i didn't notice. I've fixed them to be the same! ⭐